src/components/SearchForm.tsx

```diff
@@ -29,6 +29,7 @@ export default class SearchForm extends React.Component<SearchFormOwnProps, Sear
   render(): JSX.Element {
     let input = (
       <Input
+        id="search-input"
         key="form-input"
```

Comment:
Better to also change the key to `search-input`, or is it not worth it?

Reply:
Yeah, for consistency. Not a big deal since it's just a key for react.